Dockerfile-py2.7

```diff
@@ -25,11 +25,14 @@ ENV DISPLAY :99
 
 WORKDIR /workspace/Qt.py
 ENTRYPOINT cp -r /Qt.py /workspace && \
-    python build_caveats_tests.py && \
+    python build_caveats.py && \
+    python build_membership.py && \
 	Xvfb :99 -screen 0 1024x768x16 2>/dev/null & \
-	sleep 3 && \
+	sleep 10 && \
```

Comment:
Yes, I don't know if it's a memleak or what is going on... but I need to rebuild my containers at least once a day. And I have to restart XQuartz a few times too. So I'm not sure what's causing that.

But so 3 seconds isn't always enough... and that's why I increased it to something I don't have to go back and tweak with +1 second...